### PR TITLE
UUID to persons

### PR DESCRIPTION
**Pull request by AlexDo1**
Added the UUID logic to the Person class and updated the api (find_person(), add_person(), gut_uuid()).

Not sure if there are other instances where I should have updated something.

### CODE REVIEW

**Comment by AlexDo1** [comment]
It would be cool if I could link this PR to the issue but I think that I don´t have the permission to do that.
Is there perhaps a way to grant me such permissions without having direct write access and being able to break everything?

**Comment by mmaelicke** [comment]
I'll check that and grant you the permissionsAm 22.01.2021 17:46 schrieb AlexDo1 <notifications@github.com>:

It would be cool if I could link this PR to the issue but I think that I don´t have the permission to do that.

Is there perhaps a way to grant me such permissions without having direct write access and being able to break everything?

File: VERSION
Comment (mmaelicke) [inline VERSION:1]: Sorry, I should have been more clear on that. We increase the patch version for each PR only once. 
It is also good practice to do that on the master-branch only, and not in the PR itself. Otherwise we might run into problems if two PRs are open at the same time, as they will use the same new Version. Thus, the PR does not increment, but master increments, after the PR has been merged,

File: metacatalog/api/catalog.py
Comment (mmaelicke) [inline metacatalog/api/catalog.py:22]: See comment above. Should actually be 0.2.7

File: metacatalog/api/catalog.py
Comment (mmaelicke) [inline metacatalog/api/catalog.py:39]: While this will work (at least I assume so), it would be way better to rename the local variable `group` to i.e. `person` here for two reasons:
1. `person` describes way better what this variable is about
2. `group` has been used above. It is not considered good practice to re-use a variable in the same scope like this.

File: metacatalog/api/find.py
Comment (mmaelicke) [inline metacatalog/api/find.py:464]: see comments above and pls use `0.2.7`

File: metacatalog/models/person.py
Comment (mmaelicke) [inline metacatalog/models/person.py:29]: 0.2.7

**Review by mmaelicke — CHANGES_REQUESTED** [review]
Nice work! 
I think I caused some confusion about the versioning with my mail. Please consider my comments concerning versioning.

File: VERSION
Comment (mmaelicke) [reply VERSION]: I changed this, by resolving the merge conflict this change caused. You don't have to change anything here anymore.

File: metacatalog/api/catalog.py
Comment (AlexDo1) [reply metacatalog/api/catalog.py:39]: Sure, I didn´t see that

**Review by mmaelicke — APPROVED** [review]

**Pull request merged**
